package.json

```diff
@@ -30,7 +30,7 @@
     "test:visual:android": "loki test --targetFilter=android",
     "test:visual:update": "loki update",
     "test:visual:approve": "loki approve",
-    "test:expo": "expo doctor && ./test-expo-start.sh && ./test-expo-watch-mode.sh",
+    "test:expo": "expo doctor && node expo-test.js",
```

Comment:
Changed to a node script because the `timeout` command was not available on all systems, and the node script gives me more flexibility in writing the test.